src/Worker.Extensions.DurableTask/DurableTaskExtensionStartup.cs

```diff
@@ -58,7 +58,6 @@ public override void Configure(IFunctionsWorkerApplicationBuilder applicationBui
 
         applicationBuilder.Services.Configure<WorkerOptions>(o =>
         {
-            o.InputConverters.RegisterAt<DurableTaskClientConverter>(0);
             o.InputConverters.Register<OrchestrationInputConverter>();
```

Comment:
why did we remove this line? / why is this safe?

Reply:
That line became redundant when this was added:

https://github.com/Azure/azure-functions-durable-extension/pull/2708/files#diff-27ead41f6d3f63e1192a5c044bb8814ae8f417f872c8c1b98b85ae8277a435a2R13-R14

``` CSharp
[InputConverter(typeof(DurableTaskClientConverter))]
[ConverterFallbackBehavior(ConverterFallbackBehavior.Disallow)]
public sealed class DurableClientAttribute : InputBindingAttribute
```